framework/SupervisedLearning/ARMA.py

```diff
@@ -473,7 +479,8 @@ def reseed(self,seed):
       @ In, seed, int, new seed to use
       @ Out, None
     """
-    randomUtils.randomSeed(seed)
+    randomUtils.randomSeed(seed,engine=self.randomEng)
+    self.seed=seed
```

Comment:
Not for this PR, but in the future maybe we should consider not storing the "self.seed" on the ARMA and instead use the seed from the attached engine, since there's a redundancy of information. We can look at that in the future.